jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/language/impl/JavaLanguage.kt

```diff
@@ -486,48 +486,39 @@ class JavaLanguage : TargetLanguage {
             }
         }
     }
-    private fun functionToTypeImpl(name: String = "", func: FunctionPresentation): TypeImpl {
+    private fun functionToTypeImpl(
+        isStartFunc: Boolean = false,
+        name: String = "",
+        func: FunctionPresentation
+    ): TypeImpl {
         val typeToReturn = TypeImpl(
             shortName = classNameForStaticFunction(func.shortName),
             defaultConstructorGraphId = func.graphId,
+            visibility = func.visibility,
             inheritanceModifier = InheritanceModifier.OPEN,
         )
-        val funcToMethod = MethodImpl(
-            graphId = func.graphId,
-            containingType = typeToReturn,
+        val method = typeToReturn.createMethod(graphId = func.graphId,
```

Comment:
getOrCreate

Reply:
https://github.com/UnitTestBot/jacodb/pull/60#discussion_r1151706691
there is no "getOrCreate" in TypeImpl